### PR TITLE
Mute preview tool when there are no frames in state

### PR DESCRIPTION
**Pull request by colebillys19**
Previously, when the user opens the preview tool while there are no frames in state - the tool would just have an empty frame, slider, and 0/0 counter. I added logic to check if there are frames present, if not it will "mute" the expanded preview tool (make it opaque), prevent cursor events, and show a message for the user.

Screen Shot 2019-06-26 at 1 22 33 PM

### CODE REVIEW

File: src/components/Preview.css [hunk 1]
```diff
@@ -68,3 +73,11 @@
     opacity: 1;
   }
 }
+
+.Preview-no-frames {
+  width: 360px;
+  margin: 0 auto;
+  color: #fff;
+  font-size: 1.4rem;
```
Comment (misscoded) [inline src/components/Preview.css]: I'd recommend keeping this as the font size it is by default, but making it bold -- this mimics the error state/font we see with the toasts, keeping with the pattern that's in place.

File: src/components/Preview.js [hunk 2]
```diff
@@ -95,27 +95,29 @@ class Preview extends Component {
...
-        <div className="Preview-scrubber" data-testid="Preview-scrubber">
-          <input
-            type="range"
-            min="0"
-            max={numFrames - 1}
-            value={previewIdx}
-            onChange={this.handlePreviewUpdate}
-            disabled={!numFrames}
-            aria-label="preview frame index"
+        <div className={classNames({ 'Preview-muted': !frameIDs.length })}>
```
Comment (misscoded) [inline src/components/Preview.js]: `frameIDs.length` is already set to a `const` above called `numFrames` -- you should use that instead.

**Review by misscoded — CHANGES_REQUESTED** [review]
See comments. 

If your PR changes the appearance of the UI, you need to include a screenshot.

File: src/components/Preview.css (hunk 1 above)
Comment (colebillys19) [reply src/components/Preview.css]: Done

File: src/components/Preview.js (hunk 2 above)
Comment (colebillys19) [reply src/components/Preview.js]: Done

File: src/components/Preview.js [hunk 3]
```diff
@@ -141,6 +143,12 @@ class Preview extends Component {
         {gifProgress === 1 ? (
           <div className="Preview-progress-success">Download Successful</div>
         ) : null}
+        {!frameIDs.length ? (
```
Comment (misscoded) [inline src/components/Preview.js]: `numFrames`

**Review by misscoded — APPROVED** [review]

**Comment by misscoded** [comment]
:shipit:

**Pull request merged**